apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -3492,6 +4030,45 @@ describe('Ticket', () => {
                     expect(message[1].organization.id).toEqual(ticket.organization.id)
                 })
             })
+
+            test('do not send push message if it bulk-update assignee or executor', async () => {
+                const client = await makeClientWithProperty()
+                const executor = await makeClientWithNewRegisteredAndLoggedInUser()
+                const payload = {
+                    devicePlatform: DEVICE_PLATFORM_ANDROID,
+                    appId: APP_MASTER_ID_ANDROID,
+                }
+
+                await syncRemoteClientWithPushTokenByTestClient(executor, payload)
+
+                const [ticket1] = await createTestTicket(client, client.organization, client.property)
+                const [ticket2] = await createTestTicket(client, client.organization, client.property)
+
+                const attrs = {
+                    dv: 1,
+                    sender: { dv: 1, fingerprint: faker.random.alphaNumeric(8) },
+                    executor: { connect: { id: executor.user.id } },
+                    assignee: { connect: { id: executor.user.id } },
+                }
+
+                const updatePayload = [
+                    { id: ticket1.id, data: attrs },
+                    { id: ticket2.id, data: attrs },
+                ]
+
+                const [updatedTicket1, updatedTicket2] = await Ticket.updateMany(admin, updatePayload)
+
+                expect(updatedTicket1.executor.id).toEqual(executor.user.id)
+                expect(updatedTicket2.executor.id).toEqual(executor.user.id)
+
+                let messageCount
+                const messageWhere = { user: { id: executor.user.id }, type: TICKET_EXECUTOR_CONNECTED_TYPE }
+
+                await waitFor(async () => {
+                    messageCount = await Message.count(admin, messageWhere)
+                    expect(messageCount).toEqual(0)
+                }, { delay: MESSAGE_SENDING_DElAY })
+            })
```

Comment:
It seems that this check does not give us a guarantee that notifications are really not sent.
The worker may not have time to complete the task

but ok, let's hope that the test will help us at least a little :)

Reply:
And how much time do we need to give the worker to complete its task?)